indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,47 @@ $(document).ready(function() {
         });
     });
 
+    var threeRowsHeight = 69;
```

Comment:
Yes, but can you calculate the height instead of hardcoding it? For example, create an element with the same classes as the real one and get the height from there